chainlink-agoric/README.md

```diff
@@ -27,17 +27,19 @@ This can either be a public chain or a local chain **BUT NOT the simulated chain
 cd ..
 # Install the needed dependencies.
 agoric install
-# Start local chain implementation
+# Start local chain implementation in Docker
 agoric start --reset local-chain >& chain.log &
 ```
 
-This will run a job in the background, and when it's complete, you'll see a new docker container running. You can check it out with `docker ps`
+This will run a job in the background, and when it's complete, you'll see a new
+docker container running. You can check it out with `docker ps` or just `ps` if
+you're using the `agoric` command from the Agoric SDK.
 
 ## 3. Setup Chainlink node, external adapter, and external initiator
 
 _Note: Make sure you have cd-ed into this directory_
 
-Once you are running the Agoric local-chain, simply run:
+Once the above Agoric local-chain has started, simply run:
 
 ```bash
 ./setup
```

Comment:
When I ran `setup` it churned for a while and then said:

```
Error response from daemon: Container 55c7149d08ddbbc0e0a2b10d7f02e8194a49de19a52846df5bf1caeff778ad09 is restarting, wait until the container is running
```

Reply:
oh... I think you made changes since I cloned...

Reply:
The docker stuff seems to depend on a `host.docker.internal` mechanism that's not available on my linux box.

cf. [What is linux equivalent of "host\.docker\.internal" \- Stack Overflow](https://stackoverflow.com/questions/48546124/what-is-linux-equivalent-of-host-docker-internal/67158212#67158212)

This seemed to get me a little further:

```diff
diff --git a/chainlink-agoric/etc/network-config.json b/chainlink-agoric/etc/network-config.json
index 5264552..a551f40 100644
--- a/chainlink-agoric/etc/network-config.json
+++ b/chainlink-agoric/etc/network-config.json
@@ -1,7 +1,7 @@
 {
   "chainName": "agoric",
-  "gci": "http://host.docker.internal:26657/genesis",
+  "gci": "http://172.17.0.1:26657/genesis",
   "rpcAddrs": [
-    "host.docker.internal:26657"
+    "172.17.0.1:26657"
   ]
 }

```

Reply:
I made an issue.